canvas_modules/common-canvas/src/common-properties/controls/multiselect/multiselect.jsx

```diff
@@ -183,6 +184,7 @@ class MultiSelectControl extends React.Component {
 					titleText={this.props.tableControl ? null : this.props.controlItem}
 					helperText={this.props.control.helperText}
 					readOnly={this.props.readOnly}
+					aria-label={!this.props.control.labelVisible && this.props.control?.label?.text}
```

Comment:
Same comment as above. Aria-label isn't showing up in DOM after setting `label_visible: false`.